packages/node-opentelemetry/README.md

```diff
@@ -198,6 +198,13 @@ This can be accomplished by setting `HDX_NODE_ADVANCED_NETWORK_CAPTURE` environm
 export HDX_NODE_ADVANCED_NETWORK_CAPTURE=1
 ```
 
+If you only want to capture the header or body, we also provide two separate flags for you.
+
```

Comment:
I'd suggest to update the original description than adding new one. 
```
This can be accomplished by setting `HDX_NODE_NETWORK_HEADERS_CAPTURE`  and `HDX_NODE_NETWORK_BODY_CAPTURE` environment variables to 1.
```